text/0000-road-to-10.md

```diff
@@ -0,0 +1,62 @@
+
+## Documentation
+
+We should do a pass on our documentation and make sure that everything is up-to-date and accurate. We should also make sure that there aren't any major features that are missing documentation. This will improve the onboarding experience and allow new users to more easily discover how they can use Volta.
+
+# Timeline
+[timeline]: #timeline
+
+Given the set of outstanding features, the target for 1.0 is to have a release candidate available by Wednesday, December 16. Then, assuming there are no major problems found, the official release would come on Monday, December 21.
+
+# Critique
+[critique]: #critique
+
+The main critiques are a number of high-priority features that have _not_ been included in the 1.0 release roadmap:
+
+## Yarn 2.0 / Plug'n'Play
+
+Yarn recently released version 2.0, which comes with a number of structural changes to how Yarn works. The largest of those changes is that Yarn Plug'n'Play is the behavior by default, so to support Yarn 2.0, we would need to also support Yarn Plug'n'Play. However, Yarn Plug'n'Play itself appears to have a number of adoption issues and isn't widely used. Similarly, the transition to Yarn 2.0 has been limited within the community, so there doesn't appear to be a pressing need to support these features. Finally, we should be able to make internal changes to support Yarn 2.0 / PnP without anything that would be considered a breaking change, so it's something we can look to implement after 1.0, after the behavior stabilizes and adoption grows.
+
+## Apple Silicon Support
+
+The new Apple processors are available in laptops and showing lots of promise. Though we should support native builds on those processors, currently Volta works correctly under Rosetta 2, so this isn't a blocker for a 1.0 release. When we do add support, we also need to take into account that we may need to fall back to downloading x64 versions of Node on Apple devices, since there may not be native builds available for older versions of Node.
+
+## pnpm Support
+
+Currently, Volta only supports npm and Yarn for package managers, as they are the two most popular. [pnpm](https://pnpm.js.org/) is a relative newcomer to the space, however it is growing in adoption and we have received a few requests to support it in Volta. However, there are some technical hurdles (most notably pnpm's clobbering of `NODE_PATH` when running executables) that make it difficult to implement at the moment. While pnpm support is definitely on the near-term roadmap (likely one of the first features to be tackled after 1.0), the volume of users requesting `pnpm` is low enough that it isn't necessary as part of the 1.0 feature set.
```

Comment:
A couple slight verbiage suggestions to really emphasize that we care about supporting pnpm:

> However, there are some technical hurdles

I'd change this to "There are a few remaining technical hurdles (most notably etc etc) blocking the implementation at the moment" so it doesn't sound like we have doubts about whether we _can_ do it, just a few steps left to go, with a sense of inevitability that we'll get there.

> the volume of users requesting `pnpm` is low enough that it isn't necessary as part of the 1.0 feature set

This is a little unnecessarily dismissive. Maybe something like "While pnpm support is definitely on the near-term roadmap (likely one of etc etc), the core of Volta has stabilized to the point that we don't need to block shipping for the large number of users of other package managers."